### PR TITLE
[MIG][14.0] Migrated a brand module.

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**

### CODE REVIEW

**Review by sbejaoui — APPROVED** [review]

**Comment by sbejaoui** [comment]
@AmmarOfficewalaSerpentCS ,

The merge failed, please check [pre-commit](https://github.com/OCA/brand/runs/2728047297) and [Travis](https://github.com/OCA/brand/runs/2728047044)

**Comment by AmmarOfficewalaSerpentCS** [comment]
Okay @sbejaoui

**Comment by AmmarOfficewalaSerpentCS** [comment]
@sbejaoui , 
This PR is Ready For merge

File: brand/models/res_brand_mixin.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                    % rec.display_name
+                )
+
+    @api.onchange("brand_id")
+    def _onchange_brand_id(self):
+        for rec in self.filtered("brand_id.company_id"):
+            if rec.brand_id and rec.brand_id.company_id:
+                rec.company_id = rec.brand_id.company_id
+
+    def setup_modifiers(self, node, field=None, context=None, in_tree_view=False):
```
Comment (Freni-OSI) [inline brand/models/res_brand_mixin.py]: Check setup_modifiers method in v14

**Comment by patrickrwilson** [comment]
@AmmarOfficewalaSerpentCS Can you please increase tests to make green?

**Comment by sbejaoui** [comment]
> @sbejaoui your merge command was aborted due to failed check(s), which you can inspect on [this commit of 14.0-ocabot-merge-pr-69-by-sbejaoui-bump-nobump](https://github.com/OCA/brand/commits/560a95923d45d573fa641fd59c2658740f0a0319).
> 
> After fixing the problem, you can re-issue a merge command. Please refrain from merging manually as it will most probably make the target branch red.

@AmmarOfficewalaSerpentCS ,

There still issue with travis, I created a PR #77  to finish this one as it's needed for other PRs

**Pull request closed without merging**